WooCommerce/Classes/ViewRelated/Dashboard/Orders/LastOrderDashboardRow.swift

```diff
@@ -29,13 +31,25 @@ struct LastOrderDashboardRow: View {
                     Spacer()
 
                     VStack(alignment: .trailing, spacing: Layout.spacing) {
-                        Text(viewModel.statusDescription)
-                            .foregroundStyle(.black)
-                            .captionStyle()
-                            .padding(.horizontal, Layout.Status.hPadding)
-                            .padding(.vertical, Layout.Status.vPadding)
-                            .background(viewModel.statusBackgroundColor)
-                            .cornerRadius(Layout.Status.cornerRadius)
+                        HStack(spacing: Layout.badgeSpacing) {
+                            Text(viewModel.statusDescription)
+                                .foregroundStyle(.black)
+                                .captionStyle()
+                                .padding(.horizontal, Layout.Status.hPadding)
+                                .padding(.vertical, Layout.Status.vPadding)
+                                .background(viewModel.statusBackgroundColor)
+                                .cornerRadius(Layout.Status.cornerRadius)
+                            if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.pointOfSaleOrdersi1),
+                               viewModel.isPOSOrder {
+                                Text(viewModel.salesChannelText)
+                                    .foregroundStyle(Color(uiColor: Layout.salesChannelLabelTextColor))
+                                    .captionStyle()
+                                    .padding(.horizontal, Layout.Status.hPadding)
+                                    .padding(.vertical, Layout.Status.vPadding)
+                                    .background(Color(uiColor: Layout.salesChannelLabelBackgroundColor))
+                                    .cornerRadius(Layout.Status.cornerRadius)
```

Comment:
I wonder why such a small radius. It seems that the order list uses a larger one. IMHO, it would make sense to match the look between them.